src/GitHubExtension/DeveloperId/LoginUI/EndPage.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft Corporation and Contributors
+// Licensed under the MIT license.
+
+namespace GitHubExtension.DeveloperId.LoginUI;
+
+internal class EndPage : LoginUIPage

```

Comment:
Why is this empty?

Reply:
This is a placeholder page which never gets displayed, but is just used to store the "end" state.